src/scrapers/nytimes-upshot.js

```diff
@@ -0,0 +1,89 @@
+import Nightmare from 'nightmare';
+import Promise from 'bluebird';
+import rimraf from 'rimraf';
+import cheerio from 'cheerio';
+const fs = require('fs');
+
+import validateDoc from '../services/validations';
+import { USER_AGENT } from '../../config/constants';
+
+let nightmare = new Nightmare({show:true});
```

Comment:
We don't want to set show: true, since that'll pop up the browser window. Try running `Nightmare()` without passing the object just to be sure it doesn't need an empty object passed in.